packages/odyssey-react-mui/src/createUniqueId.ts

```diff
@@ -13,5 +13,5 @@
 // This is a random number chosen to shrink down the unique ID to an arbitrary length.
 export const uniqueIdLength = 6;
 
-export const createUniqueId = (): string =>
+export const createUniqueId = () =>
```

Comment:
Any thoughts on using the built-in `Symbol` to create unique keys/ids?  See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol

Reply:
That would work, but these are meant to be string values with go into HTML. I don't think I can `.toString()` a symbol and get a clean output. If so, then I should use that instead.

Reply:
yeah, good catch -- if those go into HTML then Symbols aren't the right approach.

Reply:
Depending on the volume, Symbol() can create lots of trash since it is essentially an object wrapping a primitive. If we don't need it, let's not.